ci/jenkins/Jenkins_steps.groovy

```diff
@@ -896,24 +896,6 @@ def test_unix_python3_mkldnn_nocudnn_gpu() {
     }]
 }
 
-def test_unix_python3_tensorrt_gpu() {
```

Comment:
@mseth10 I guess this is the cause of the CI job issue

Reply:
@Kh4L you'd need to get rid of this too if you wanna remove this test:
https://github.com/apache/incubator-mxnet/blob/9981e847fff9270268385068c5b7d0c3929e46f9/ci/jenkins/Jenkinsfile_unix_gpu#L53

Reply:
that's right, if we remove this test we should also remove it from jenkins pipeline

Reply:
in master, it's removed in this commit
https://github.com/apache/incubator-mxnet/commit/c1098aa33d6795f84a19601d0319d5bb8e19f317#diff-0c2b22569b87fa79153336ea3ac94f33

Reply:
Thanks for investigating, I missed that file...